tests/integration/src/dfx_orbit/manage_settings.rs

```diff
@@ -0,0 +1,159 @@
+use crate::{
+    dfx_orbit::{
+        setup::{
+            dfx_orbit_test, setup_counter_canister, setup_dfx_orbit, setup_dfx_user,
+            DfxOrbitTestConfig,
+        },
+        util::{permit_change_operation, set_auto_approve_on_change},
+    },
+    setup::{setup_new_env, WALLET_ADMIN_USER},
+    utils::{submit_request, user_test_id},
+    TestEnv,
+};
+use dfx_orbit::{
+    args::{RequestArgs, RequestArgsActions},
+    canister::{RequestCanisterActionArgs, RequestCanisterArgs, RequestCanisterUpdateSettingsArgs},
+};
+use station_api::{
+    AllowDTO, CreateExternalCanisterOperationInput,
+    CreateExternalCanisterOperationKindAddExistingDTO, CreateExternalCanisterOperationKindDTO,
+    ExternalCanisterPermissionsCreateInput, ExternalCanisterRequestPoliciesCreateInput,
+    RequestOperationInput,
+};
+use std::time::Duration;
+
+/// Test that adding and removing controllers works
+#[test]
+fn change_controllers() {
+    let TestEnv {
+        mut env,
+        canister_ids,
+        ..
+    } = setup_new_env();
+
+    let canister_id = setup_counter_canister(&mut env, &canister_ids);
+
+    let (_, _) = setup_dfx_user(&env, &canister_ids);
+
+    permit_change_operation(&env, &canister_ids);
+    set_auto_approve_on_change(&env, &canister_ids);
+
+    // Register the canister with the station
+    submit_request(
+        &env,
+        WALLET_ADMIN_USER,
+        canister_ids.station,
+        RequestOperationInput::CreateExternalCanister(CreateExternalCanisterOperationInput {
+            kind: CreateExternalCanisterOperationKindDTO::AddExisting(
+                CreateExternalCanisterOperationKindAddExistingDTO { canister_id },
+            ),
+            name: String::from("counter"),
+            description: None,
+            labels: None,
+            permissions: ExternalCanisterPermissionsCreateInput {
+                calls: vec![],
+                read: AllowDTO {
+                    auth_scope: station_api::AuthScopeDTO::Restricted,
+                    user_groups: vec![],
+                    users: vec![],
+                },
+                change: AllowDTO {
+                    auth_scope: station_api::AuthScopeDTO::Restricted,
+                    user_groups: vec![],
+                    users: vec![],
+                },
+            },
+            request_policies: ExternalCanisterRequestPoliciesCreateInput {
+                change: vec![],
+                calls: vec![],
+            },
+        }),
+    );
```

Comment:
let's please wrap this into a helper

Reply:
After talking to @keplervital , it turns out that it is possible to remove the requirement for the registration altogether. This was implemented in #396. It allowed me to remove the code entirely.